layers/+distributions/spacemacs-base/funcs.el

```diff
@@ -979,3 +979,17 @@ a split-side entry, its value must be usable as the SIDE argument for
                                             text-scale-mode-amount) 1)
                                   (if (car (window-margins))
                                       (car (window-margins)) 1)))))
+
+(defun spacemacs/linum-enabled-p ()
+  "Return non-nil if line numbers should be enabled for current buffer.
+Decision is based on `dotspacemacs-line-numbers'."
+  (if dotspacemacs-line-numbers
+      (if (or (string-match-p "\\*.*\\*" (buffer-name))
+              (and (plist-get dotspacemacs-line-numbers :size-limit-kb)
+		   (> (buffer-size) (* 1000 (plist-get dotspacemacs-line-numbers :size-limit-kb)))))
+	  nil
+        (if (plist-get dotspacemacs-line-numbers :enabled-for)
+            (member major-mode (plist-get dotspacemacs-line-numbers :enabled-for))
```

Comment:
Just read some docs about `member` using `equal` and `memq` using `eq` being stricter, but am still blurry about the implications.